gtsam/discrete/DiscreteConditional.h

```diff
@@ -57,6 +57,29 @@ class GTSAM_EXPORT DiscreteConditional: public DecisionTreeFactor,
   /** Construct from signature */
   DiscreteConditional(const Signature& signature);
 
+  /**
+  * Construct from key, parents, and a Table specifying the CPT.
```

Comment:
Same deal here about the CPT. We know what the context is, but so should other people. :)

Reply:
Good point